### ISSUE TITLE
SVGO not found

### ISSUE THREAD

**Issue by luke-j** [1000ch/Sublime-svgo#15]
I'm getting the following error when running a SVGO command:

```
svgo
Error: internal/modules/cjs/loader.js:818
throw err;
^

Error: Cannot find module '/Users/luke/Library/Application Support/Sublime Text 3/Packages/SVGO/svgo.js'
at Function.Module._resolveFilename (internal/modules/cjs/loader.js:815:15)
at Function.Module._load (internal/modules/cjs/loader.js:667:27)
at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:60:12)
at internal/main/run_main_module.js:17:47 {
code: 'MODULE_NOT_FOUND',
requireStack: []
}
```

The package is looking for the `.js` file and not able to reconcile it with the `.mjs` file. My node version is `v14.5.0`.

**Comment by simonfoxenano** [comment]
![image](https://user-images.githubusercontent.com/85542316/132278521-24c6378d-7b7a-4f88-890d-5be416e16224.png)

**Comment by zvuc** [comment]
Get the same error with the latest version. FYI I have not been able to use this plugin for some time already (at least more than a year as far as I can recall...) because it always threw some error when I wanted to run the command, so I had to use other manual ways to minify SVGs :( 

Really wish there will be a fix soon that will finally make this work because it will save a great amount of time in my day to day workflow!!

**Comment by giorgiofnool** [comment]
me too!

**Comment by 1000ch** [comment]
@luke-j @simonfoxenano @zvuc @gfnool sorry for bothering you, I believe this seems to be fixed on the latest version (`v4.4.0`).
